CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+### 0.3.0
+
+FEATURES:
+* [32](https://github.com/nginxinc/nginx-prometheus-exporter/pull/32): Add nginxexporter_build_info metric.
+* [31](https://github.com/nginxinc/nginx-prometheus-exporter/pull/31): Implement nginx_up and nginxplus_up metrics. Add -nginx.timeout cli argument for setting a timeout for scrapping metrics from NGINX or NGINX Plus.
+
+UPGRADE:
+* Use the 0.3.0 image from our DockerHub: `nginx/nginx-prometheus-exporter:0.3.0`
+* Download the latest binaries from [GitHub releases page](https://github.com/nginxinc/nginx-prometheus-exporter/tree/v0.3.0).
```

Comment:
Fix `GitHub release page` link so it links to releases page rather than git branch
```suggestion
* Download the latest binaries from [GitHub releases page](https://github.com/nginxinc/nginx-prometheus-exporter/releases/tag/v0.3.0).
```

Could also just link to https://github.com/nginxinc/nginx-prometheus-exporter/releases if preferred